.github/workflows/ci.yml

```diff
@@ -77,11 +78,15 @@ jobs:
             - name: Set output variables
               id: bake_metadata
               env:
-                  BAKE_METADATA: ${{ steps.bake.outputs.metadata }}
```

Comment:
Does this mean that the 'bake' step could be removed entirely?

Reply:
The step `id` can be removed, but the baking step is the actual step for building the image.